docs/0002-N-tuple-tree.md

```diff
@@ -0,0 +1,180 @@
+# OCFL Community Extension 0002: N-tuple Trees for OCFL Storage Hierarchies
+
+  * Authors: Neil Jefferies
+  * Minimum OCFL Version: 1.0
+  * Obsoletes: n/a
+  * Obsoleted by: n/a
```

Comment:
I still feel a little uneasy that extensions aren't versioned. Without a version, the only safe way to introduce breaking changes to an extension is by creating a new extension.

Reply:
Thoughts on mechanisms?

Reply:
Punting to a separate issue #4